extensions/adobe/b2b/marketo/marketo-web-url.schema.json

```diff
@@ -45,7 +45,7 @@
       "$ref": "#/definitions/web-url"
     }
   ],
-  "meta:status": "experimental",
```

Comment:
Did we ever use any marketo schema for B2B ? I guess this schema is a very old one and was never intended to use. Could you please check and confirm ?

Reply:
@mjasrotia yes we use this in our b2b schema.  You can see this here in our [documentation](https://experienceleague.adobe.com/docs/experience-platform/sources/connectors/adobe-applications/marketo/marketo-namespaces.html?lang=en )